src/common/components/accountBenefitsModal/accountBenefitsModal.tpl.html

```diff
@@ -0,0 +1,17 @@
+<div class="col-xs-12">
```

Comment:
Do you think it would be easy to use the same html and/or component as the accountBenefits component? I guess the button would have to change. Is this modal for use outside of the thank you component too?